### ISSUE TITLE
[BUG] dependency info properties file contains error messages

### ISSUE THREAD

**Issue by jlowe** [NVIDIA/spark-rapids#1031]
**Describe the bug**
When building as a normal developer, the various *-info.properties files for each shim contains error messages rather than properties:
```
$ jar xvf dist/target/rapids-4-spark_2.12-0.3.0-SNAPSHOT.jar spark-3.0.0-info.properties
 inflated: spark-3.0.0-info.properties
$ cat spark-3.0.0-info.properties

/home/jlowe/src/rapids-plugin-4-spark/build/dependency-info.sh: line 30: /jenkins/printJarVersion.sh: No such file or directory
/home/jlowe/src/rapids-plugin-4-spark/build/dependency-info.sh: line 33: /jenkins/printJarVersion.sh: No such file or directory
$
```

**Steps/Code to reproduce bug**
Build with `mvn verify` and execute the commands from the shell session snippet above.

**Expected behavior**
The properties file should contain valid key=value entries.  If the shell script that generates the properties file encounters an error then the build should fail.

**Comment by revans2** [comment]
@jlowe @NvTimLiu is this still a problem or has this been fixed?

**Comment by jlowe** [comment]
Fixed by #1095
